pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/stats/ConsumerStatsImpl.java

```diff
@@ -84,7 +85,11 @@
     @JsonIgnore
     private int clientVersionLength;
 
+    // ignore this json field to skip from stats in future release. replaced with readable #getLastAckedTime().
+    @Deprecated
```

Comment:
Is it ok to keep the timestamp? 
It's useful to calculate time used, like how long since it's last acked.